prow/bump.sh

```diff
@@ -115,18 +124,21 @@ elif [[ -z "$cmd" || "$cmd" == "--list" ]]; then
       exit 1
     fi
   fi
+}
+
+if [[ "$cmd" == "--push" ]]; then
+  push
+  exit 0
+elif [[ -z "$cmd" || "$cmd" == "--list" ]]; then
+  list
 elif [[ "$cmd" =~ v[0-9]{8}-[a-f0-9]{6,9} ]]; then
   new_version="$cmd"
+elif [[ "$cmd" == "--latest" ]]; then
+  new_version="$(list-options 1)"
```

Comment:
maybe check that `new_version` is nonempty?

Reply:
Seems like `list-options` already exits non-zero unless it finds something?